lib/crypto.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+namespace SCCP\Crypto;
+
+class HashAlgo extends Enum {
+    private const SHA1 = 0;
+    private const SHA256 = 1;
+}
```

Comment:
We currently don't have an Enum class. We would either need to create/import one, or just forgo  the Enum extension.